README.md

```diff
@@ -1,99 +1,99 @@
+Open the [inventory file](ansible/inventory) - each machine that will be joining the Kubernetes cluster must be defined as either a master or worker node. To leverage the highly available topology configuration you would ideally have 3 masters available as a minimum, otherwise 1 master node is fine, however, it won't be highly available.
 
-### Master Node(s)
+> Note: Ensure the `hostname` matches what the machine was given when flashed the SD card and `ansible_host` matches the IP allocated to the host on your prefered subnet.
 
-The master node configuration is copied from your machine to the Raspberry Pi with the following command:
+When the inventory has been configured with all the hosts that will be joining the Kubernetes cluster we can run the following command to verify SSH connectivity can be established.
 
 ```bash
-RPI_HOSTNAME=k8s-master-01 KUBE_NODE_TYPE=master RPI_IP=192.168.1.101 make deploy
+ansible all -m ping -i inventory -u pi
 ```
 
-Once the deployment has completed you should be now able to manually `ssh` to the newly configured master node using the folling command:
+A successful response should look something like the following:
 
-```bash
-ssh pi@k8s-master-01.local
+```diff
++ k8s-worker-01 | SUCCESS => {
++    ...
++    "changed": false,
++    "ping": "pong"
++    ...
++}
 ```
 
-Kubernetes should be running and you can view the syslogs with the following command:
+> Note: If your output returns success for each ping then you can continue, otherwise there may be some misconfiguration of either the inventory file, or network connectivity issues.
 
-```bash
-tail -f /var/log/syslog
-```
+There are a variety of different configurable options in the Ansible automation. These options can be located in the [vars.yml](ansible/vars.yml) file. In this file you have the option to change the container runtime ("CRI") and the container network interface ("CNI") that is used within the cluster.
 
-Additionally you can verify your nodes status is `READY` with the following command:
+Below is a matrix of the currently supported options.
 
-```bash
-kubectl get nodes
-```
+**Container Runtime**
 
-Now that the master node is running successfully you can either add more master nodes to the cluster or create worker nodes within the cluster. To be able to connect additional nodes into the cluster we will need to extract the kube config and the join token output by the master node. This can be done with the following command:
+| Name | Supported |
+| ---- | --------- |
+| Docker | Yes |
+| ContainerD | Yes |
 
-```bash
-RPI_HOSTNAME=k8s-master-01 KUBE_NODE_TYPE=master RPI_IP=192.168.1.101  make post-install
-```
+**Container Network Interface**
 
-The kube config and join token should now be in the ./output directory and will be used in configuring additional nodes.
+| Name | Supported |
+| ---- | --------- |
+| Weave | Yes |
+| Flannel | Yes |
 
```

Comment:
link to the file or a snippet showing how the different options can be configured

Reply:
currently link to vars file do you think i should go more specific and show it here as well? 

```
[vars.yml](ansible/vars.yml)
```